go/chat/convsource.go

```diff
@@ -783,9 +783,27 @@ func (p *pullLocalResultCollector) String() string {
 	return fmt.Sprintf("[ %s: t: %d ]", p.Name(), p.num)
 }
 
+func (p *pullLocalResultCollector) realHits() (total int) {
```

Comment:
maybe add an `minimum int` and break if we hit that total so we don't loop across all results?

Reply:
or make return a `bool` for if we hit anything real

Reply:
Hmm, yeah this makes sense, will do!